mobile/src/main/AndroidManifest.xml

```diff
@@ -107,11 +107,13 @@
             android:label="@string/widget_type_color"
             android:showWhenLocked="true"
             android:exported="false"
+            android:taskAffinity=".ui.ColorItemActivity"
             tools:targetApi="o_mr1" />
         <activity
             android:name=".ui.SelectionItemActivity"
             android:showWhenLocked="true"
             android:exported="false"
+            android:taskAffinity=".ui.SelectionItemActivity"
```

Comment:
In Octodroid I used `android:excludeFromRecents="true" android:taskAffinity=""` for a similar case (URL handler activity). I think we definitely should add the former here as well, and I also think 'don't group into any other activity' is a bit more correct than grouping into itself, because I think the latter becomes similar to the `singleTop` or `singleTask` launch modes.